lib/logflare_web/templates/layout/navbar.html.heex

```diff
@@ -77,6 +77,7 @@
     </ul>
     <%= if assigns[:user] do %>
       <ul class="navbar-nav ml-auto">
+        <.team_switcher :if={assigns[:teams] && length(assigns[:teams]) > 1} team_context={Logflare.Teams.TeamContext.build(assigns)} teams={assigns[:teams]} current_path={@conn.request_path} />
```

Comment:
Might be better to include the current query params as well, in case user is switching on the query page.

Reply:
all makes sense. will do.